04/app.js

```diff
@@ -1,8 +1,11 @@
 document.addEventListener('DOMContentLoaded', init);
+const colorInput = document.querySelector('input:nth-of-type(1)');
+const rangeInput = document.querySelector('input:nth-of-type(2)');
 
 function init() {
     const boxElement = document.querySelector('.box');
     setBoxShadow(boxElement, '#000000');
+    document.querySelector('section').addEventListener('change', () => setBoxShadow(boxElement, colorInput.value, rangeInput.value/100));
```

Comment:
Spróbuj też tak to rozwiązać, że przy zmianie wartości dla `range` w czasie rzeczywistym (przy przesuwaniu suwaka) również widać od razu efekt.